internal/controllers/core/kubernetesapply/reconciler.go

```diff
@@ -346,8 +347,40 @@ func (r *Reconciler) runYAMLDeploy(ctx context.Context, spec v1alpha1.Kubernetes
 }
 
 func (r *Reconciler) runCmdDeploy(ctx context.Context, spec v1alpha1.KubernetesApplySpec) ([]k8s.K8sEntity, error) {
-	// TODO(milas): implement Cmd apply
-	return nil, errors.New("Cmd apply not supported")
+	cmd := model.Cmd{
+		Argv: append([]string(nil), spec.Cmd.Args...),
+		Dir:  spec.Cmd.Dir,
+		Env:  append([]string(nil), spec.Cmd.Env...),
+	}
+
+	timeout := spec.Timeout.Duration
+	if timeout == 0 {
+		timeout = v1alpha1.KubernetesApplyTimeoutDefault
+	}
+	ctx, cancel := context.WithTimeout(ctx, timeout)
+	defer cancel()
+
+	var stdoutBuf bytes.Buffer
+	runIO := localexec.RunIO{
+		Stdout: &stdoutBuf,
+		Stderr: logger.Get(ctx).Writer(logger.WarnLvl),
```

Comment:
i wouldn't use WarnLvl here...WarnLvl is more for discrete warnings that we want the UI to call attention to. Stderr is generally more informational. Let's use InfoLvl